File: problems/g/grading/grading.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <iostream>

using namespace std;

int main() {
    int a, b, c, d, e, score;

    if (scanf("%d %d %d %d %d\n", &a, &b, &c, &d, &e) != 5) {
        printf("Failed to read thresholds!");
    }

    cin >> score;

    if (score >= a) {
        cout << "A";
    }
    else if (score >= b) {
        cout << "B";
    }
    else if (score >= c) {
        cout << "C";
    }
    else if (score >= d) {
        cout << "D";
    }
    else if (score >= e) {
        cout << "E";
    }
    else {
        cout << "F";
    }
}

File: problems/g/grading/grading.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    a, b, c, d, e = [int(val) for val in input().split()]
    score = int(input())

    if score >= a:
        print("A")
    elif score >= b:
        print("B")
    elif score >= c:
        print("C")
    elif score >= d:
        print("D")
    elif score >= e:
        print("E")
    else:
        print("F")



if __name__ == '__main__':
    main()